hvpy/api_groups/jpeg2000/tests/test_get_jp2_header.py

```diff
@@ -23,3 +26,17 @@ def test_error_handling():
 def test_url_property():
     params = getJP2HeaderInputParameters(id=9838343)
     assert params.url == "https://api.helioviewer.org/v2/getJP2Header/"
+
+
+def test_set_api():
+    params = getJP2HeaderInputParameters(id=9838343)
+    assert params.url == "https://api.helioviewer.org/v2/getJP2Header/"
+
+    hvpy.set_api("https://api.beta.helioviewer.org/")
+    params = getJP2HeaderInputParameters(id=9838343)
+    assert params.url == "https://api.beta.helioviewer.org/getJP2Header/"
+
+    os.environ["URL"] = "https://localhost:3000/"
+    hvpy.set_api()
```

Comment:
So I have to do this to change the URL?

Reply:
We decided this in the meeting on, 14th [here are the notes](https://github.com/Helioviewer-Project/python-api/wiki/Meeting-Notes#14-july-830-ist) and (pseudo code) by Daniel.

Reply:
I see, thanks for the context. 

I still don't understand why a person should need to run `hvpy.set_api()` if they have set a OS environment.

Reply:
Yes, I know. I tried to do what you said but I was not able to unless I ran the `set_api` function.